examples/nodejs/eventbridge/infrastructure/lib/eventbridge-stack.ts

```diff
@@ -0,0 +1,220 @@
+import * as cdk from "aws-cdk-lib";
+import { Construct } from "constructs";
+import * as dynamodb from "aws-cdk-lib/aws-dynamodb";
+import * as events from "aws-cdk-lib/aws-events";
+import * as pipes from "aws-cdk-lib/aws-pipes";
+import * as iam from "aws-cdk-lib/aws-iam";
+import * as logs from "aws-cdk-lib/aws-logs";
+import * as sqs from "aws-cdk-lib/aws-sqs";
+import {Secret} from "aws-cdk-lib/aws-secretsmanager";
+
+export class EventbridgeStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const momentoApiKeyParameter = new cdk.CfnParameter(this, 'MomentoApiKey', {
+      type: 'String',
+      description: 'The API key for Momento.',
+    });
+
+    const gameScoreDemoTable = new dynamodb.Table(this, "game-scores-demo-table", {
+      tableName: "game-scores-demo",
+      partitionKey: { name: "GameId", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "GamerTag", type: dynamodb.AttributeType.STRING },
+      stream: dynamodb.StreamViewType.NEW_AND_OLD_IMAGES,
+      removalPolicy: cdk.RemovalPolicy.DESTROY,
+    });
+
+    const logGroup = new logs.LogGroup(this, "AccessLogs", {
```

Comment:
I might move some of the less interesting stuff to a helper function, to shorten up this constructor for readability. Stuff like the IAM permissions and the log group, maybe even the api key param/secret.